Components/Alerts/alert.css

```diff
@@ -0,0 +1,23 @@
+.alert{
+    display: flex;
+    align-items: center;
+    width: 95%;
+    max-width: 1000px;
+    height: 55px;
+    margin: 25px 10px 25px 10px;
+    padding-left: 20px;
+    border-radius: 5px;
+    font-size: 1.2rem;
+}
+.alert-error{
+    background-color: rgb(247, 59, 59);
```

Comment:
Since you will need these colors further. You can add these colors to root variable.